packages/drivers/routerlicious-driver/src/tokens.ts

```diff
@@ -32,21 +37,21 @@ export interface ITokenResponse {
 export interface ITokenProvider {
     /**
      * Fetches the orderer token from host.
+     *
      * @param tenantId - Tenant ID.
      * @param documentId - Optional. Document ID is only required for document-scoped requests.
      * @param refresh - Optional flag indicating whether token fetch must bypass local cache.
```

Comment:
@ssimic2 This is the parameter I was recommending we make mandatory. I would like to add some docs here noting that it will be made mandatory in a future release (+ associated breaking.md entries), but figured I would check first and make sure that's safe for us to do.

Reply:
I don't have strong preference here, but I would add two items that I see more important given that we don't need to wait for next breaking cycle to make improvements:

1. Whether this is mandatory or not, we need to make sure to document the behavior and for consumers to understand when (and how) FF is using this field. The flag name may look self-exaplanatory, but we should still document with some sort of (simple) statement around refresh flag being set to true if previous async call failed with auth (401) error. This is one way to highlight and indirectly say "think about this, its important".
2. For any faults/failures in the system our telemetry and/or user facing errors should indicate if this flag was set to true or false. Im not sure if that is true currently, but we should double check.

@zhenmichael may have more feedback here as he is working on few TokenProvider improvements as well.

Reply:
With the upcoming telemetry changes, it won't explicitly state that the refresh param is true. However, it does provide an error message: "Host-provided token fetcher threw an error". While this implies the refresh param is set to true, it may be more clear to explicitly state it.

Reply:
I'll add some notes. I'll also add notes that the params are going to be mandatory in the future, and will plan on doing that in next.